### ISSUE TITLE
Project 235 Receiver Data Uploads Produce Error

### ISSUE THREAD

**Issue by leberrigan** [MotusDev/Motus-TO-DO#480]
See https://sgdata.motus.org/status?jobID=441502

Upload completes successfully, only produces error during processing. Tried re-zipping and uploading again with no success. 

Does anyone get notified by these errors?

**Comment by Hussell** [comment]
I was looking at this yesterday, but haven't figured it out yet. It looks as though it may be something to do with the large number of files in the upload. Try uploading an archive with roughly the first 1/3rd of the files?

Users get an email. No-one here does, but I've been looking at the jobs list every day for the past few months and dealing with the easy-to-understand-and-fix errors.

**Comment by Hussell** [comment]
Argh, there's a comment in the code right where the error happens:

>requires that sum of path lengths of unique top-level directories don't exceed the shell's command-line buffer

So the problem is indeed the large number of files/days/receivers in that upload. I noticed there's data from several receivers in that upload. Please try uploading each receiver's data separately.

**Comment by leberrigan** [comment]
Thanks - I'll let the user know. Should be good to close this one. I'll reopen it if they're still having trouble.

**Comment by Hussell** [comment]
Thomas uploaded the files from one of the receivers, and got "There were no new files in the dataset, so I didn't do anything." I tried rerunning the upload job, which didn't work. I tried rerunning the receiver, but there is a mismatch between the db and the file repo (files are listed in the database but aren't in the file repository). I think I may have to delete the files listed in the database to clean up the partial work done by the original job, which errored-out at a particularly annoying point.
